src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Running locally
+section: Basics
+position: 3
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a project, you can start developing your app locally.
+You can use the `space dev` command to start a local development server for your project.
+
+```bash
+space dev --open # start a local development server and open the app in your browser
+```
+
+You can control what command is run for each micro by editing the `dev` field in your `Spacefile`.
+
+```yaml
+v: 0
+micros:
+  - name: client
+    src: ./client
+    engine: svelte
+    primary: true
+    path: client
+  - name: api
+    src: ./api
+    engine: nodejs16
+    dev: node index.js
+    path: api
+```
+
+## Running micros individually
+
+Optionally you can instead use separate commands to run each micro individually, then use a reverse proxy to route requests to the correct micro.
+
```

Comment:
This is very unclear and confusing.

Reply:
you are right, I tried to reuse the example above to make it clearer.